var/spack/repos/builtin/packages/flux-core/package.py

```diff
@@ -46,10 +52,10 @@ class FluxCore(AutotoolsPackage):
     depends_on("py-cffi", type=('build', 'run'))
     depends_on("py-six", type=('build', 'run'), when="@0.11.0:,master")
     depends_on("py-pyyaml", type=('build', 'run'), when="@0.11.0:,master")
-    depends_on("py-jsonschema", type=('build', 'run'), when="@master")
+    depends_on("py-jsonschema", type=('build', 'run'), when="@0.12.0:,master")
```

Comment:
```suggestion
    depends_on("py-jsonschema", type=('build', 'run'), when="@0.12.0:")
```
Now that #1983 has been merged, `master` > `0.12.0`.